env_vars.md

```diff
@@ -12,52 +12,54 @@ In this file you can find an explanation of all the enviroment variables that ar
 * `ALLOW_ORIGIN` - Sets the `Access-Control-Allow-Origin` header to the specified value.
 
 ### Optional (without defaults)
-* `MAIL_SERVICE` - For bulk add operations, the email service to use for sending progress reports.
-* `MAIL_HOST` - Email service host (usually SMTP config).
-* `MAIL_ACCOUNT` - Sender email account address.
-* `MAIL_PASSWORD` - Sender email account password.
-* `OAUTH2_TOKEN_URI` - Endpoint of OAuth2 token service
-* `OAUTH2_PUBLIC_KEY`- OAuth2 service public key used to encrypt / verify tokens
-* `OAUTH2_CLIENT_ID`- GraphQL server OAuth2 Client ID
-* `GRAPHIQL_REDIRECT_URI`- RedirectURI of the graphiql-auth client, used to migrate the default keycloak OAuth2 service
-* `SPA_REDIRECT_URI`- RedirectURI of the single-page-app client, used to migrate the default keycloak OAuth2 service
+* `OAUTH2_TOKEN_URI` - Endpoint of OAuth2 token service.
+* `OAUTH2_PUBLIC_KEY`- OAuth2 service public key used to encrypt / verify tokens.
+* `OAUTH2_CLIENT_ID`- GraphQL server OAuth2 Client ID.
+* `GRAPHIQL_REDIRECT_URI`- RedirectURI of the graphiql-auth client, used to migrate the default keycloak OAuth2 service.
+* `SPA_REDIRECT_URI`- RedirectURI of the single-page-app client, used to migrate the default keycloak OAuth2 service.
 
 ### Optional (with sensible defaults)
 * `ERROR_LOG` - Debug logs verbosity. Can be either "verbose" or "compact". Default value is `compact`.
 * `EXPORT_TIME_OUT` - Maximum amount of time in milliseconds before the server throws a timeout error when exporting data. Default is `3600`.
 * `LIMIT_RECORDS` - Maximum number of records that each request can return, default value is `10000`.
-* `PORT` - The port where the app is listening, default value is `3000`
+* `PORT` - The port where the app is listening, default value is `3000`.
 * `POST_REQUEST_MAX_BODY_SIZE` - Maximum size of the GraphQL request in MB. Default is `1mb`.
 * `MAX_TIME_OUT` - Maximum number of milliseconds that a zendro server will wait to connect with another zendro server. Default value is `2000`.
 * `REQUIRE_SIGN_IN` - Boolean to toggle the required sign in to the graphql server. Default is `true`.
 * `SALT_ROUNDS` - Number of salt rounds when hashing a new password. Default is `10`.
+* `WHITELIST_ROLES` - Whitelist an array of roles to grant permission regardless of user roles.
+* `DOWN_MIGRATION` - Allow down migration of a model. Executing the down migration will delete all records and its associations.
```

Comment:
whitelist: the env var is a string, for example "reader,editor", it would be parsed as an array in graphql-server. But it is a string variable. You can also add the string example here.
down_migration: by default it is false, in this case user can perform down migration if the table or collection is empty, namely no records. When this env var is true, user can drop all data and metadata for non-empty table.